### PR TITLE
AOM-140: Update post data sent during OWA installation

### PR DESCRIPTION
**Pull request by malmike**
## JIRA TICKET NAME:
[AOM-140: Update post data sent during OWA installation](https://issues.openmrs.org/browse/AOM-140)
### SUMMARY:
Update the data sent in the post request for OWA installation based on the changes made for [OWA-70](https://issues.openmrs.org/browse/OWA-70)

### CODE REVIEW

**Comment by patlub** [comment]
Looks good to me

**Review by aenkya — APPROVED** [review]
Looks good

File: app/js/components/manageApps/ManageApps.jsx [hunk 1]
```diff
@@ -619,7 +619,6 @@ export default class ManageApps extends React.Component {
     if (addon && addon.appDetails.type === 'OWA') {
       this.requestUrl = '/owa/installapp';
       let urlObject = {
-        'fileName': addon.appDetails.name,
```
Comment (dkayiwa) [inline app/js/components/manageApps/ManageApps.jsx:622]: What is the reason behind this change? The ticket description is not clear.

Comment (malmike) [reply app/js/components/manageApps/ManageApps.jsx:622]: I have updated the ticket description.

**Comment by Annettesunday** [comment]
Looks fine
